docs/setting-up/client/index.md

```diff
@@ -331,29 +333,47 @@ dnf localinstall *.rpm
 
 5. Run the installer.
 
+    !!! caution alert alert-warning "Non root users"
+        Run this command before next step:
+        ```sh
+        export PMM_DIR=YOURPATH
+        ```
+        where YOURPATH replace with you real path, where you have required access.  
+
+        **Node, MySQL and PostgreSQL exporters wont be avalaible.**
+
+    This command require root permissions (if you skipped step for non root users).
     ```sh
     ./install_tarball
     ```
 
 6. Change the path.
 
     ```sh
-    PATH=$PATH:/usr/local/percona/pmm2/bin
+    PATH=$PATH:$PMM_DIR/bin
     ```
 
-7. Set up the agent
+7. Set up the agent (pick right command for you depends on permissions)
 
+    !!! hint "Root permissions"
     ```sh
     pmm-agent setup --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml --server-address=192.168.1.123 --server-insecure-tls --server-username=admin --server-password=admin
     ```
 
-8. Open a new terminal and run the agent.
+    !!! caution alert alert-warning "Non root users"
+    ```sh
+    pmm-agent setup --config-file=${PMM_DIR}/config/pmm-agent.yaml --server-address=192.168.1.123 --server-insecure-tls --server-username=admin --server-password=admin --paths-tempdir=$PMM_DIR/tmp --paths-exporters_base=$PMM_DIR/exporters --paths-node_exporter=$PMM_DIR/exporters/node_exporter --paths-mongodb_exporter=$PMM_DIR/exporters/mongodb_exporter --paths-postgres_exporter=$PMM_DIR/exporters/postgres_exporter --paths-proxysql_exporter=$PMM_DIR/exporters/proxysql_exporter --paths-azure_exporter=$PMM_DIR/exporters/azure_exporter --paths-pt-summary=$PMM_DIR/tools/pt-summary --paths-pt-pg-summary=$PMM_DIR/tools/pt-pg-summary --paths-pt-mongodb-summary=$PMM_DIR/tools/pt-mongodb-summary --paths-pt-mysql-summary=$PMM_DIR/tools/pt-mysql-summary
+    ```
+
+
+
+8. Run the agent.
 
     ```sh
-    PATH=$PATH:/usr/local/percona/pmm2/bin pmm-agent --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml
+    pmm-agent --config-file=${PMM_DIR}/config/pmm-agent.yaml
```

Comment:
PMM-DIR is not defined for root user, so how this step goes for ppl with root perm ?

Reply:
Here is default in script for PMM-DIR value:
PMM_DIR=${PMM_DIR:-/usr/local/percona/pmm2}

So it is not needed for root users.

Reply:
does it exports PMM_DIR? it just runs.. to import it's vars you probably need to do something like `. ./install_tarball`/`source ./install_tarball`

Reply:
I mean the vars stay in context of the script, if you aren't starting new term within the script...

Reply:
Good point, you are right. Checkout updated doc and screenshot here: https://jira.percona.com/secure/attachment/28761/Screenshot%20from%202021-07-07%2013-28-47.png. Thanks for notice that.